exams/pearson/download_test.py

```diff
@@ -278,7 +278,9 @@ def setUpTestData(cls):
         cls.now = datetime.now(pytz.utc)
         cls.processor = download.ArchivedResponseProcessor(sftp)
         with mute_signals(post_save):
-            cls.success_profiles = ExamProfileFactory.create_batch(2)
+            cls.success_profiles = ExamProfileFactory.create_batch(2) + [
+                ExamProfileFactory.create(profile__student_id=1000),  # disjoint id and student_id
```

Comment:
kind of a nitpick, but shouldn't you set `profile__id` _and_ `profile__student_id` if you want them to be explicitly disjoint?